serilog/Program.cs

```diff
@@ -26,11 +22,37 @@ static void Main()
             {
                 var customer = customerGenerator.Generate();
                 var order = orderGenerator.Generate();
-                
+
                 logger.Information("{@customer} placed {@order}", customer, order);
-                
+
                 Thread.Sleep(1000);
             }
         }
+
+        private static LoggerConfiguration GetLogConfiguration()
+        {
+
+            // note: use of ElasticsearchJsonFormatter is optional (but recommended as it produces 'idiomatic' json)
+            // If you don't want to take a dependency on 'Serilog.Formatting.Elasticsearch' package
+            // you can also other json formatters such as Serilog.Formatting.Json.Serilog.Formatting.Json.JsonFormatter
+
+            bool.TryParse(Environment.GetEnvironmentVariable("USE_LOGSPOUT"), out var useLogspout);
+            if (!useLogspout)
+            {
+                // log direct to logstash
+                return new LoggerConfiguration()
+                    .WriteTo.DurableHttpUsingFileSizeRolledBuffers(
+                        requestUri: "http://logstash:31311",
+                        textFormatter: new ElasticsearchJsonFormatter(),
+                        batchFormatter: new ArrayBatchFormatter())
+                    .WriteTo.Console();
+            }
```

Comment:
So if we're proposing to create a new repo showing logspout as the method for sending to logstash, then this sink should be removed, and the conditional logic for selecting it should be removed - right?

Reply:
Correct, that's the reason I think it should be a new repository and not cause confusion in a sample of Serilog.Sinks.Http.

I'll get you the repo with configured Elastic Stack in a few hours, or a empty repository for you to fill right now if you are up for it.